docs/interactions/Slash_Commands.md

```diff
@@ -742,6 +782,31 @@ Edits a followup message for an Interaction. Functions the same as [Edit Webhook
 
 Deletes a followup message for an Interaction. Returns `204` on success.
```

Comment:
Would it be worth it to specify that's it's a 204 status code? I'm not sure if this is necessary, but I wanted to give the idea.

Reply:
I think consistency in our docs just has `Returns <number>`